### ISSUE TITLE
It's possible to add  remove column feature ?

### ISSUE THREAD

**Issue by hungcrush** [nhn/tui.grid#1045]
As the title, can we add removeColumn feature? Or in the future do we have the plan for it? 😬 😬

**Comment by js87zz** [comment]
@hungcrush 
Yes, we have the plan for the feature you commented as providing API. 
However, the exact schedule is not decided for other priorities.
If grid provide this feature, I will comment here. Thank you!

**Comment by hungcrush** [comment]
@js87zz I'm trying to do that feature. But I'm stuck when I appended new Column, data.rawData is not watching anymore.

```javascript
rawData.forEach((row, rowIndex) => {
    row = observable({ ...row, [newColumnName]: '' });
    viewData[rowIndex].valueMap = observable({
      ...viewData[rowIndex].valueMap,
      [newColumnName]: {
        className: '',
        disabled: false,
        editable: true,
        formattedValue: '',
        invalidStates: [],
        value: ''
      }
    });
```

If you have time, could you let me khow what should I do to refresh observable... I really need that feature.. 

Thank you.

**Comment by js87zz** [comment]
@hungcrush 
How about using the [setColumns](https://nhn.github.io/tui.grid/latest/Grid#setColumns) API?
The API is inefficient to you, but I think It's better to use this API before developing the API like `removeColumn`. After developing it, you can replace with it. How about it?

**Comment by hungcrush** [comment]
@js87zz 
Hi.. I so glad when see you reply... I did it, I use partialObservable to append observe column data. 😬
My demo:
![ezgif-5-bff0c73cb110](https://user-images.githubusercontent.com/8098916/82201851-287ac780-992b-11ea-8436-dac272bfe89e.gif)

**Comment by js87zz** [comment]
@hungcrush 
It's well done~!Great! :)
If you have enough time, it's good to give us PR on your API.
Thank you!

**Comment by hungcrush** [comment]
@js87zz 
Sure.. I will create a pull request soon... I want to learn a lot of things from TUI GRID 
The observable system is very very awesome...  

Thank you so much
